doc/api/v0.yaml

```diff
@@ -441,3 +464,40 @@ components:
               type: array
               items:
                 $ref: "#/components/schemas/Proposal"
+
+    ReviewTag:
+      type: string
+      enum:
+        - Alignment
+        - Verifiability
+        - Feasibility
+        - Impact
+        - Auditability
+
+    Review:
+      properties:
+        id:
+          type: integer
+          format: i32
+        proposal_id:
+          type: integer
+          format: i32
+        rating_given:
+          type: integer
+          format: i32
+        assessor:
+          type: string
+        note:
+          type: string
+        tag:
+          type: ReviewTag
+
+    AdvisorReviews:
+      allOf:
+        - $ref: "#/components/schemas/Review"
+        - type: object
+          properties:
+            proposals:
+              type: array
+              items:
+                $ref: "#/components/schemas/Review"
```

Comment:
I don't get it. How can this result object have properties from `Review` and also a `proposals` array with other `Review` objects?

Reply:
This is a copypasta error.